monai/apps/detection/transforms/GeometricTransforms.py

```diff
@@ -0,0 +1,217 @@
+from typing import List, Tuple
+from typing import List
+import cv2
+import json
+import numpy as np
+from monai.transforms import Compose, ScaleIntensity, EnsureChannelFirst, \
+    Flip, Rotate90, RepeatChannel, AsChannelLast, LoadImage, ToNumpy, ToTensor, Transpose, \
+    Resize, RandAdjustContrast, GaussianSmooth
+
+
+def sanity_check(image, labels):
+    """
+    plot the label on the image to make sure the label is correct
+    Args:
+        image_np:
+        labels:
+
+    Returns:
+    """
+    img = np.ascontiguousarray(image, dtype=np.uint8)
+
+    if labels.shape[0] > 2:
+        print("Label: ", labels)
+        new_labels = np.vstack([labels, labels[0]])
+        for i in range(0, len(new_labels)-1):
+            print("Label: ", new_labels[i], new_labels[i+1])
+            img = cv2.line(img, (int(new_labels[i][0]), int(new_labels[i][1])),
+                             (int(new_labels[i+1][0]), int(new_labels[i+1][1])),
+                          (255, 255, 255), 2)
+    else:
+
+        img = cv2.line(img, (int(labels[0][0]), int(labels[0][1])),
+                         (int(labels[1][0]), int(labels[1][1])),
+                         (255, 255, 255), 2)
+
+    return img
+
+
+class ImageIntensityAndAnnotation:
```

Comment:
I think all the MONAI transforms should inherit and follow the base signature, @ericspod @atbenmurray what do you think?

Thanks.